### PR TITLE
New package: git-interactive-rebase-tool-2.1.0.

### PR DESCRIPTION
**Pull request by jgarte**
Home page: https://gitrebasetool.mitmaro.ca/

General

    [x] This is a new package and it conforms to the quality requirements

Have the results of the proposed changes been tested?

    [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me

Does it build and run successfully?

    [x] I built this PR locally for my native architecture, (x86_64)

### CODE REVIEW

**Comment by jgarte** [comment]
Hi, I opened a new pr because I lost the old tree I was working on when I originally submitted this and wasn't sure how to integrate it into the old PR via the github web UI.

Tests are passing.

This currently builds successfully for me and I tested it. I'm using the package on a daily basis.

File: srcpkgs/git-interactive-rebase-tool/template [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Template file for 'git-interactive-rebase-tool'
+pkgname=git-interactive-rebase-tool
+version=2.1.0
+revision=1
+archs="i686 x86_64"
```
Comment (paper42) [inline srcpkgs/git-interactive-rebase-tool/template:5]: Why was this line added again?

Comment (jgarte) [reply srcpkgs/git-interactive-rebase-tool/template:5]: hmmm, I can remove it

Comment (jgarte) [reply srcpkgs/git-interactive-rebase-tool/template:5]: I think @ericonr had requested it for building on `i686`

Comment (paper42) [reply srcpkgs/git-interactive-rebase-tool/template:5]: > There shouldn't be a reason to do this, but if there is, it should be clearly documented.

if it's not specified, the package will be built for all architectures. This way you restricted it to 2 architectures and excluded their musl variants.

Comment (jgarte) [reply srcpkgs/git-interactive-rebase-tool/template:5]: Oh @ericonr had requested to remove it also :)

Comment (jgarte) [reply srcpkgs/git-interactive-rebase-tool/template]: > > There shouldn't be a reason to do this, but if there is, it should be clearly documented.
> 
> if it's not specified, the package will be built for all architectures. This way you restricted it to 2 architectures and excluded their musl variants.

Is there a way I can test building for all architectures on my local machine with xbps-src?

Comment (tornaria) [reply srcpkgs/git-interactive-rebase-tool/template]:
```
$ ./xbps-src -m masterdir.musl binary-bootstrap x86_64-musl
$ ./xbps-src -m masterdir.musl build git-interactive-rebase-tool
$ ./xbps-src -m masterdir.musl check git-interactive-rebase-tool
```

**Comment by jgarte** [comment]
Here's an explanation for why those particular 3 tests might be failing:

https://github.com/MitMaro/git-interactive-rebase-tool/issues/573

I'll look into how to set that directory as read only.

**Comment by ericonr** [comment]
Our CI runs as root, which might break those tests.

**Comment by jgarte** [comment]
> Our CI runs as root, which might break those tests.

@ericonr Do you suggest I disable those three tests then?

**Comment by ericonr** [comment]
If possible, yes.

**Comment by jgarte** [comment]
> If possible, yes.

@ericonr 

I skipped those three tests in the template. Everything is green now :)
